package.json

```diff
@@ -53,7 +53,14 @@
       "fast-loops@<1.1.4": ">=1.1.4",
       "fast-xml-parser@<4.4.1": ">=4.4.1",
       "elliptic@<6.5.7": ">=6.5.7",
-      "micromatch@<4.0.8": ">=4.0.8"
+      "micromatch@<4.0.8": ">=4.0.8",
+      "webpack@>=5.0.0-alpha.0 <5.94.0": ">=5.94.0",
```

Comment:
Same comment as below, it's in our devDeps, so let's update it there directly and remove it from overrides

Reply:
Fixed